### DISCUSSION TITLE
imgview show filenames

### DISCUSSION THREAD

**Discussion by qrockz** [jarun/nnn#1857] [Q&A] [answered]
How can I achieve this:
![68747470733a2f2f692e696d6775722e636f6d2f313469504449712e706e67](https://github.com/jarun/nnn/assets/56853194/2b31a57f-a868-4f65-bfa2-aa707678e1d7)

I want to see the filename under the thumbnail.

Using imgview looks like this on my system:
![now](https://github.com/jarun/nnn/assets/56853194/08a47e70-b11e-437b-a112-1705cf1fd1ea)

**Comment by jarun** [comment]
That must be a different image thumbnail viewer tool, not the plugin. I've forgotten which one I had used.

**Comment by qrockz** [comment]
@jarun ohh come on :smile:

**Comment by jarun** [comment, marked as answer]
I remembered - [lsix](https://github.com/hackerb9/lsix)
